### PR TITLE
add lock file, env config file

### PR DESCRIPTION
**Pull request by SergeyKovalchuk**

### CODE REVIEW

**Comment by Guria** [comment]
why yarn
shouldn't it be a team decision?

**Comment by SergeyKovalchuk** [comment]
Let's discuss it, PULL REQUEST --- It's just request)
Would you like to discuss yarn vs npm topic?

File: app/.env [hunk 1]
```diff
@@ -0,0 +1,5 @@
+PROXY_PATH=http://evbyminsd6293.minsk.epam.com:8080/
```
Comment (Guria) [inline app/.env]: Can't see where this config is used. Don't we need introduce smth like `dotenv` to have effect?

File: app/.nvmrc [hunk 2]
```diff
@@ -0,0 +1 @@
+6.10
```
Comment (Guria) [inline app/.nvmrc]: `npm i node@6.10.0` ?
https://www.npmjs.com/package/node
https://github.com/aredridel/node-bin-gen

File: app/.env (hunk 1 above)
Comment (avarabyeu) [reply app/.env]: guys, please, no any EPAM server names in config files

**Comment by Guria** [comment]
I guess it could be closed now, since dotenv was already implemented

**Comment by DiscoElevator** [comment]
Closed in favor of #672

**Pull request closed without merging**